<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskBooleanParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskBooleanParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxNewSolid">
     <property name="toolTip">
      <string>Make new shape using the tool features without base feature.</string>
     </property>
     <property name="text">
      <string>New shape</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkboxDeleteOnRemove">
     <property name="text">
      <string>Delete tool features on remove</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string>Tool features</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetBodies">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboType">
     <item>
      <property name="text">
       <string>Fuse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Common</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Compound</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Section</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBodyAdd</tabstop>
  <tabstop>buttonBodyRemove</tabstop>
  <tabstop>listWidgetBodies</tabstop>
  <tabstop>comboType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
